pom.xml

```diff
@@ -832,6 +832,7 @@
               <exclude>nbproject/**</exclude>
               <exclude>nb-configuration.xml</exclude>
               <exclude>nbactions.xml</exclude>
+              <exclude>CONTRIBUTING.md</exclude>
```

Comment:
You can add the license header using markdown comments (i.e  `<!-- comment -->`) and not have exclude this file from RAT.  See the `README.md`